### PR TITLE
Invoke cert cleanup

### PR DESCRIPTION
**Pull request by roncain**

### CODE REVIEW

File: src/System.Private.ServiceModel/tools/setupfiles/CleanupCertificates.cmd [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+for /f "delims=" %%A in ('where /F /R tests CertificateCleanup.exe') do set "CertCleanupVar=%%A"
+if '%CoreRunVar%' EQU '' (
+  Echo Could not cleanup certificiates because was unable to locate CertificateCleanup.exe under %~dp0..\..\..\..\bin
+  goto done
+)
+
+echo Cleaning up certificates using: %CoreRunVar% %CertCleanupVar%
+%CoreRunVar% %CertCleanupVar%
+
```
Comment (roncain) [inline src/System.Private.ServiceModel/tools/setupfiles/CleanupCertificates.cmd]: Reviewers: this technique is a little unorthodox and comments are welcome.  We need to find where CoreRun.exe and CertificateCleanup.exe were placed by the build, but we don't have access to the MSBuild properties that determine their full path.  This CMD file just locates where they landed and executes them.  I will create a similar .sh script for Linux in a separate PR.

**Comment by iamjasonp** [comment]
I've been playing with this one, but can't really figure out a better way to hook this in for the time being... :frowning: 

LGTM unless we find a better solution.

**Comment by iamjasonp** [comment]
@dotnet-bot test outerloop please

**Comment by iamjasonp** [comment]
Buddy built this - at least we know this is running and removing certs properly! 

```
WCF for .NET Core test certificate cleanup tool
https://github.com/dotnet/wcf

Removes test certificates generated by the Bridge during testing

Checking StoreName 'My', StoreLocation 'CurrentUser'
  FE8655A777EE3CAAE07A474242B57DA489641502. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  FE8204AA5EC9E4AB6B925A52A6C9A75CB06BC86F. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  F4B2BB57A6276343526A08BA54F50A8AE13B3144. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  EEC9CF79DBDCAA1731AF3864E2438F90A55606D0. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  EAE32D6E2D498AC67F1106B4A40211790A1F5E96. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  DFDC59C43B58334E2EA3A356949F25B72FC8545E. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  DF07C626C19A5AA59F818A0C526BD17D6AAF8806. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  D9CC20F9C667DE9952ECA64DBFDE3168481E5F46. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  D6720E12537D31A6FCF521E2AEE7AA817FF8C562. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  D3693927B9253ABF2086A761651E2580671340D2. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  D0A43EEDB403D29EA01F832649D324B5E8B01C44. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  CA40B60ED082239437AA476A826A9E5A3653A9C0. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  C12C57EAD963DD09806689E786E160B270644919. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  BAB0EB4D5273D04E014EBADA09B47D0D7EE889FC. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B9ADE8384356BDBF84FE2EDB464A41381959CFD7. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B181BC303F3665DFF373347E9A6904A8AE2AC75C. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B0417B1F2754FFC8BF9EFC8DDBF5AA7F3EF3FDDF. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  ABDB044547BA7E4897AC98BDB0F96902C5AC2EE2. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A80C7665755E61B6DBB2931093F45DBE8AA7A824. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A6614B213A7DD19D63B3625C7B447FB0B5F9405D. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A308AAE87B0703B670A65BA5052E37676A676473. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A303DABE5C62BEB6E2E3A86F5CA7D73F59E60862. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9D5D6F064A7C6E78ACC792B907CE303856A34CB3. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9C82F1B47D2B9B8859352E304460116C6FECC35E. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9C67AB80A5D65542056EF4A008DD7940F5334262. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9C56B93EB88FE84657605A92A727E7465F32218B. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9C1FC3AC69D88797D88501DCA303D0DC94088E28. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  93763188E3726E3838A4AFC94A5910922F4C2C75. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9350FA1EBE9EADF9AE4C210C012FB0A6AD390AF6. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  8C7688842181E75A1C12361B8E34673E4BA162FC. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  89965329C907A1ABCE179AB6043679A800DD53AB. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  86B3D11EB25D832EB087AEC2F61D95AC6FE7E95A. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  8578E20BB75B07B535A3EF320AD8598F6C0C6437. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  83F8E39EED14A624F169415178C7B0519C46BA49. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  826D3CD238090C326817C0563A21F4A03A32A384. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7FE069F17D282999A2394A363DC4FC9DDAFEB973. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7D2476397CE849AA4D75E4F0DC0B55BA5E87B999. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7653669FB4C6193F7A1D300E392CFB4EEA2F4E4F. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  764C0D7E692F1118E7DC7C6B6358E6C7D18FC052. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  71A0F6A00C8CE37EF2E3560FDBF50AEB0653CD2F. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  70EF5F06C00181D28A37BF2A0A3329E021BAC5E4. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  6AFB3109B51C2A9AA8F5BF4A9483066205299FDF. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  67ECECF56822FC83EFF7F7BA72CECE90AAD587A1. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  6506ADFD7EC588ED40B14064919A9960820DAC62. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  616444038F7F87CF6AD86084D89C1D7CFC87A7F4. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  589E7FA4A40A8661B153B22A836471A96D596D40. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  57FA3F695868941FDDBA0FCA5013D26F114DD3FD. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  56C701F2426481A78EF2E080CB93499B0AE5AC8F. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  5514C9950782A8DB22C00FE995EAECD37CDFCDE1. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  52182024F30BDCCA8A7A5CF20CB074BB12851BE1. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  5048EDB5E27881B395D01C935924E0EEF23756B6. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  485A86FB928BD03E3F472CB30EEB0350872D465B. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  3FFF1E770706670C74F7F36D2B69AEB68BD20301. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  3B4A403D4A6A027DA00BBC3935F81446BF99E167. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  30F3E0A1831BEAF88536CE98801724EE343BAF3B. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2CAFC27D2E70B104108D302944EF235483F114F3. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2C0A7D665DCAD018FD27960C7B8AAF9F4DF5FC39. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2278FB7DD448E9A37920AAB04BF7E3DE8F6FBB73. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  1F1A5995D3B7ED0A1E1AC94D99FA6869EE3230BB. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  1E898B89770018A8A9320440B9228880D6AFDE7B. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  1C4563CD808899E55215537118C19EF2337B80B0. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  1ADFEC8CBFAAA80C6608463D99B104C50675E041. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  1615DB0DBA7846C810D48FA4E7CFD5CCD413B5BA. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  0CBA9CB438844F264172FD6A455CA5F86A4CA35A. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  0A909749FB4C05D583CD755C4EF460D2C484DFCC. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  09412C1A0C2948FED357A85C5275737C0E60E300. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  0539AFB5CB0110CC9843D7C3E51A527B71181955. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  03C9564EAD61C3E68F7E505EC92A9FAEE1303856. Subject: 'CN=WCF Client Certificate, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
68 certificates removed

Checking StoreName 'My', StoreLocation 'LocalMachine'
  FF515BB1743078AAA7119BA234C8DE376A5095B0. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  FD544AE2F527083C98B738BDC777E270A46C7E39. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  F3A092E5C7CC172A6604EEE4CFAC3902A3D3E2FE. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  F284F36A254767C343454D403FF2D117EC13FCA3. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  EF2B02829DD0D34EA5D106DED3AC9302D40D0237. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  E98FB4819B28EFC25BC7BCE72F192B1DB0611735. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  E8BFC307A20A4757A7260F5D2593632E8072DB77. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  E5E7863DBF79CC77A193ECF74CAC57AFB962508A. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  E283A77DDFF2ABC9E62AFC83E8F84BF3D5360867. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  DF301B741AFA956A6902DD28B6AD8DE6DB739314. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  D7EA1D9FDD78FF37348FB182C6D9AC3CBEA24152. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  D58ED2DB9A5F613CA30071940F44B79CB779EB58. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  CDFB69D43F6117A1E18A96698367B1597D359A37. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  CDB16BA8D8B8856CA3F55336F50D0FD48FC8847A. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  CDB00CF86C1212DA1E6BBC241007298DC3A671F8. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  C6E87B3CAB11362092AE60EE0D8B8340AB3356FF. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  C5D492AD1D275BC4837D8CE5B459E498D10E284F. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  C24B1D5FE78075A3D4E560501A5FF25FA1770124. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  C0081D10AAD3D4598C03D94C2A40F32BEAABDD27. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  BF03A4A0761CA650974C7537E55856F9F7D822D3. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  BC3FCB1F64135E8CDB1285A15B98095B590DC580. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  BA94B144440025030E0DBC587859A7CDF6B5CC78. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B99D197D9D063AB7E7F9791F4BDD66FDDFA97670. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B85B5E1F10210DCB61FE8152381177BF289F4C06. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B452BC91676B517FAC3C50B4BADD9949F8422BD9. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  B182494AFE05E0A1DA777E67C273A75538BB115C. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A48BF398A71E6E810159F958CA9738F79BA0B541. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A3E74E6790754C4213A296293D202C94478E5EE7. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  A095DF62DAC7C06AB7C622C80941D6E700CD2720. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  9DFC245378BCD33289074CA4E71B46908E781170. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  97E7AEFFBF791D371C4BACB94DB72A80C57F7F72. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  8B664AC41F114EA199774CC4B036B4F7C8FEAC68. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  8ADF0ADC36F4CB327A445474D148D736C4AFE663. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  8720C8A0032487EE3E296DBCF68B627FF9E2D334. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  83ED2AAB2AB5C8B39086B990829F0170C0696D3E. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  81275F63789F26FE48D0438CA76913673C91D632. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7FB31FFEE6D552E7B684984C84EAE0FC2DFFCE24. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7F6236A254C8A61C7BE61ABCAC45D6B31AF6AF3B. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7E27B4FB6FD18AACEF34F9A0E65068C18F9DF489. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7893E1CC8D0B4CF75B530BFA19045D5FA772B531. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  77E6759A58901D40C484775BABBF02AA459E546A. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  76350139A9E287FB9A3D8393AB8F9F13C91DCD40. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7550D95830B279EAB84B0C51935567C59533B960. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  7048A11C86ACB7221430A75A9692C58FFE1FFEF7. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  6C8AE4AAFA29D07125E0071C7DB70607DDAB15E0. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  69ED075A769E945608A3719413E13AC1E170BDC9. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  644DC51BED0052AB8CFA8AC546F89951A360DF9B. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  60E7BE3145A6061BFA271C38F0F36692201C7582. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  5F9177D123558259F3BEA29FFF4600A9603683A8. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  5BA87D035BE7E21CF0277A370AAEE52E1B6D46B6. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  575C7A8524696B0EBD3C49787BA97BF10674A4FE. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  498130AD28F6234959D092A9746F412562C95931. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  48292AAA9BC830681FF7EEEF60BE4B19C0D41211. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  40BFCA3473BF846DE85563EB2D5AAB163F31C522. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  40AC7EE88B90D59F119A17076F4FA09F8BA2B738. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  3B193D75769B7DEAC217FE45506BFAE5A322D714. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  379350D34ACFC08999620F67D8BDA3F86BC3E6CD. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  373FB50947E496365C4C87F5C237360A163647F9. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  3565A84343FBCE5E8ADDFBB4B3386FFFA65485C0. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  34A60BD3082E140462182CF236F6D765E197A2FE. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  3499633CB782A6FB257C07AA1000230FA48A85A8. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  310EECA286C0B85A0F9AFF28BF9BBEC112AFA930. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  30965114CA94CC9BD625D815E1B5445B5B0AEA6C. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  30104732B2E8E5DEB3DF3214C147B3A3F67BBA8F. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2FEB6D94F67DAE65C7E2550F5F22774E4E0FC8D5. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2E873A7960AE37DE1EB43BE0E89ED0A95D81FD61. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2DE47EEEACF0C9CAD8E41454EB199FE7A6863DF8. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2DE438DC5FA16326907C2F8FDC2D8215ECC1F7A9. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  2A0B7703C5CD2DBBA1593FDF7B144BD60F4AA2A8. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  20F88FD18335B10F17CD18327ACE1F0A1E3A002A. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  17D7BD53569598220A8012861D2053F1EBD0EBE5. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  16B3BEC703BA6734520FC7586082B1573A3AABFA. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  1657EF73799E3EBB96A4B3EF3FB418B54AFFC739. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  15AA1925161382BAA6EC0949858B90B0D070F614. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  144A07C3D175D38468BB4D2D35E5ED1F06585E1F. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  11DF4F66079AF74151010A9529E3A0B9A03A54EC. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  10B65EF3580FEB5BB097E6E328E682E3B70CEFF0. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  0E5CC62960D5ECDD1E804C8F3EDD9F1B867D60E4. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  0DE8E1A4C4216A8F2C8E5E5756968DA8766A183B. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  09BF4901E8DF8803437D59BD1B5C68FA788F5E1D. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
  0650D4AC521513289A607A9EB4C9096C696A2D32. Subject: 'CN=dci-win-fbld-1, O=DO_NOT_TRUST, OU=Created by https://github.com/dotnet/wcf' ... removed
81 certificates removed

Checking StoreName 'Root', StoreLocation 'LocalMachine'
0 certificates removed

Checking StoreName 'Root', StoreLocation 'CurrentUser'
0 certificates removed
```

**Pull request merged**